src/test/java/api/support/MultipleJsonRecords.java

```diff
@@ -53,4 +55,9 @@ public int totalRecords() {
   public JsonObject getFirst() {
     return records.get(0);
   }
+
+  @Override
+  public Iterator<JsonObject> iterator() {
+    return records.iterator();
+  }
```

Comment:
Why is an iterator needed? What does it provide over a `forEach` method?

Reply:
Iterator has been removed.